Universal Editor docs for merchants and developers (COMDOX-1471) (#747)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchants/quick-start/create-content.mdx

> 1. **Document Authoring (DA.live)**: The content management app (CMS) where your storefront's pages, blocks, and assets are authored and managed.

Reviewer: Link to sections --  so step 1  -- bolded text links to DA.live section, ...

### DA.live (Document Authoring)

> DA.live (Document Authoring) is the content management system (CMS) you use to build you storefront site. It is where you create and manage your storefront's pages, blocks, and assets. DA.live provides a collaborative environment for content teams to create and structure site content, manage translations, and control publishing workflows. It integrates with the Universal Editor for in-context editing and with Edge Delivery Services for fast, reliable content delivery. DA.live supports versioning, localization, and the features required to manage multiple commerce storefronts.

Reviewer 2: ```suggestion
DA.live (Document Authoring) is the content management system (CMS) you use to build you storefront site. It is where you create and manage your storefront's pages, blocks, and assets. DA.live provides a collaborative environment for content teams to create and structure site content, manage translations, and control publishing workflows. It can integrate with the Universal Editor for in-context editing and with Edge Delivery Services for fast, reliable content delivery. DA.live supports versioning, localization, and the features required to manage multiple commerce storefronts.
```

Learn more about DA.live in the <Link href="https://www.aem.live/developer/da-tutorial" text="DA.live tutorial" />.

### Universal Editor

The Universal Editor enables business users and content authors to edit website content in real-time with immediate live previews and publishing. This in-context editing experience eliminates the need to switch between different tools or wait for publishing cycles to see changes.

> Learn how to use the Universal Editor in the [Using the Universal Editor](/merchants/quick-start/universal-editor/) topic.

Reviewer: ```suggestion
Learn more about the Universal Editor by visiting the [Using the Universal Editor](/merchants/quick-start/universal-editor/) topic.
```
